.github/workflows/create-react-app-typescript-ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     name: Playwright tests
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
This one is a little bit redundant because `./.github/actions/playwright` already does it. That playwright action is separated for the reusability but does it even have to be? Is there any specific reason why it's not inlined here?